tests/torch/data/reference_graphs/quantized/hawq/inception_v3_device_VPU__mode_liberal__ratio_1.5__with_target_scopes.dot

```diff
@@ -1,926 +1,730 @@
 strict digraph  {
```

Comment:
@ljaljushkin please take a look at the change in this file and help ascertain whether my understanding of what should be checked in a target scopes test for HAWQ is correct.